index.js

```diff
@@ -220,6 +220,41 @@ module.exports = {
 				}))
 				.catch(err => console.error(err));
 		},
+		/**
+		 * Get the aggregate exchange totals based on time periods.
+		 */
+		aggregate({ timeSeries = '1d', max = 30 } = {}) {
+			let formattedEntity;
+			if (timeSeries === '1d') {
```

Comment:
[np] you can use an object for that `entityMap = { '1d': 'dailyTotals', '15m': 'fifteenMinuteTotals' }